app/themes/DefaultTheme/composer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "swp/default-theme",
```

Comment:
this should be configured like: 

```
{
    "name": "swp/default-theme",
    "title": "Superdesk Web Publisher default theme"
}
```

we dont use composer to install theme so "extra" property is not needed.